js/aax.js

```diff
@@ -2280,6 +2281,213 @@ module.exports = class aax extends Exchange {
         return result;
     }
 
+    parsePosition (position, market = undefined) {
+        //
+        //    {
+        //        "autoMarginCall": false,
+        //        "avgEntryPrice": "3706.03",
+        //        "bankruptPrice": "2963.3415880000",
+        //        "base": "ETH",
+        //        "code": "FP",
+        //        "commission": "0.02964824",
+        //        "currentQty": "2",
+        //        "funding": "-0.04827355",
+        //        "fundingStatus": null,
+        //        "id": "385839395735639395",
+        //        "leverage": "5",
+        //        "liquidationPrice": "2983.07",
+        //        "marketPrice": "3731.84",
+        //        "openTime": "2021-12-31T18:57:25.930Z",
+        //        "posLeverage": "5.00",
+        //        "posMargin": "14.85376824",
+        //        "quote": "USDT",
+        //        "realisedPnl": "-0.07792179",
+        //        "riskLimit": "10000000",
+        //        "riskyPrice": "3272.25",
+        //        "settleType": "VANILLA",
+        //        "stopLossPrice": "0",
+        //        "stopLossSource": 1,
+        //        "symbol": "ETHUSDTFP",
+        //        "takeProfitPrice": "0",
+        //        "takeProfitSource": 1,
+        //        "unrealisedPnl": "0.51620000",
+        //        "userID": "3829384",
+        //        "ts": 1641027194500
+        //    }
+        //
+        const contract = this.safeString (position, 'symbol');
+        market = this.safeMarket (contract, market);
+        const size = this.safeString (position, 'currentQty');
+        let side = undefined;
+        if (Precise.stringGt (size, '0')) {
+            side = 'buy';
+        } else if (Precise.stringLt (size, '0')) {
+            side = 'sell';
+        }
+        const leverage = this.safeString (position, 'leverage');
+        const unrealisedPnl = this.safeString (position, 'unrealisedPnl');
+        const currentQty = this.safeString (position, 'currentQty');
+        const contractSize = this.safeString (market, 'contractSize');
+        const initialQuote = Precise.stringMul (currentQty, contractSize);
+        const avgEntryPrice = this.safeString (position, 'avgEntryPrice');
+        const initialMargin = Precise.stringMul (initialQuote, avgEntryPrice);
+        const marketPrice = this.safeString (position, 'marketPrice');
+        const notional = Precise.stringMul (initialQuote, marketPrice);
```

Comment:
@frosty00 I tried to calculated `initialMargin` and `notional`. Are these calculations correct?

Reply:
i think notional is ok but the initialMargin is wrong

Reply:
also since they give you the bankrupt price and the liquidationprice you can calculate the maintenance margin

Reply:
> also since they give you the bankrupt price and the liquidationprice you can calculate the maintenance margin

Whats the formula

Reply:
@frosty00 How do I calculate maintenance margin from the bankrupt price and the liquidation price?

Reply:
@kroitor I still don't know how to calculate the maintenance margin from those two variables

Reply:
@frosty00 I tried to calculate the maintenance margin, is that how it's calculated?